### ISSUE TITLE
Add Documentation for Working with Playwright

### ISSUE THREAD

**Issue by ahaywood** [redwoodjs/redwood#11318]

**Comment by ahaywood** [comment]
@joshdotjs Is it worth adding a mini guide / tutorial on how to create a e2e test? -- Or maybe this is better as a blog post that we link out to?

**Comment by Josh-Walker-GM** [comment]
Yeah sounds like a great idea! A reference, a tutorial which adds some more context/example around the usage and then getting some blog content out of that tutorial example. Sounds win-win.
